google/cloud/spanner_v1/batch.py

```diff
@@ -36,15 +37,18 @@ class _BatchBase(_SessionWrapper):
     """Accumulate mutations for transmission during :meth:`commit`.
 
     :type session: :class:`~google.cloud.spanner_v1.session.Session`
-    :param session: the session used to perform the commit
+    :param session: The session used to perform the commit.
+
+    :type mutations: list
+    :param mutations: The list into which mutations are to be accumulated.
     """
 
     transaction_tag = None
     _read_only = False
 
-    def __init__(self, session):
+    def __init__(self, session, mutations=[]):
         super(_BatchBase, self).__init__(session)
-        self._mutations = []
+        self._mutations = mutations
```

Comment:
Revert the changes for `_BatchBase`. There is no need to pass mutations to this and set it.
```suggestion
        self._mutations = []
```